### PR TITLE
Mac: Redraw immediately to prevent shaking on window resize

### LINKED ISSUE #1837: Resize event is not emitted immediately/synchronously
When the window is resized, it looks like the resize event is not actually emitted immediately, but put into a queue to be dealt with later. This causes issues trying to draw to winit windows: https://github.com/rust-windowing/glutin/issues/1340 , https://github.com/gfx-rs/wgpu/issues/1168 .

### PR DESCRIPTION
**Pull request by xiaopengli89**
Fixes #1837, gfx-rs/wgpu-rs#536, rust-windowing/glutin#1340, servo/webrender#1640

### CODE REVIEW

**Comment by ArturKovacs** [comment]
At a first glance this seems to be related to or a duplicate of #1812, but I'm going to give a proper review later.

**Comment by ArturKovacs** [comment]
I was wrong, this addresses different problem than #1812.

**Review by ArturKovacs — CHANGES_REQUESTED** [review]
This looks perfect, but please add an entry to the `CHANGELOG.md` file, something like:
```
- On macOS, emit `RedrawRequested` events while the window is being resized.
```

**Comment by xiaopengli89** [comment]
Alright, and thanks for your review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 - Added `WindowBuilder::with_position` to allow setting the position of a `Window` on creation. Supported on Windows, macOS and X11.
 - Added `Window::drag_window`. Implemented on Windows, macOS, X11 and Wayland.
 - On X11, bump `mio` to 0.7.
+- On macOS, emit `RedrawRequested` events while the window is being resized.
```
Comment (maroider) [inline CHANGELOG.md:21]:
```suggestion
- On macOS, emit `RedrawRequested` events immediately while the window is being resized.
```
Based on the PR title, linked issues, and the tiny amount of code I've seen in the macOS backend, this seems like it describes the changes better.

**Pull request merged**